### PR TITLE
fix(Config/Code): Correct typo

### PR DESCRIPTION
**Pull request by AnchyDev**
## Changes Proposed:
- Fix a typo with the MyPlayerAcoreString enum.
- Set the default module enable config to 0 (the description states the default is 0).

## Issues Addressed:
- N/A

## SOURCE:

- N/A

## Tests Performed:

- Tested the module still works as expected.

## How to Test the Changes:

1. Enable the module in the config.
2. Start server.
3. Login to a character.
4. Observe there is a system message sent from the module.

### CODE REVIEW

**Review by Helias — APPROVED** [review]

File: conf/my_custom.conf.dist [hunk 1]
```diff
@@ -14,4 +14,4 @@
 #                     1 - Enabled
 #
 
-MyModule.Enable = 1
+MyModule.Enable = 0
```
Comment (Helias) [inline conf/my_custom.conf.dist:17]: I would put it as 1 by default, because the developer that will use this, will surely set it as 1 to work on it

**Comment by pangolp** [comment]
I agree with Helias, it is better to leave it at 1, so that the message is displayed. As for the typo error, it was probably my fault. Thanks for correcting it.

**Comment by Helias** [comment]
thanks for the PR!

**Comment by pangolp** [comment]
> > I agree with Helias, it is better to leave it at 1, so that the message is displayed. As for the typo error, it was probably my fault. Thanks for correcting it.
> 
> Yeah fair enough, I was mostly annoyed by the typo every time I created by a new module I had to fix it. 😄

That probably happened when I did the part of adding the translations to the DB. Create the enum, and obviously, I misspelled it and I forgot. As generally, that part is usually deleted, to create the new module, it happened over time. But thanks for correcting it.

**Pull request merged**